docker-compose.yml

```diff
@@ -51,32 +58,38 @@ services:
         soft: -1
         hard: -1
     restart: always
-
   logstash:
     image: "docker.elastic.co/logstash/logstash:${TAG_LOGSTASH}"
-    depends_on:
-      - elasticsearch
+    container_name: logstash
     volumes:
       - "./configuration/logstash/logstash.conf:/usr/share/logstash/pipeline/logstash.conf:ro"
       - "./configuration/logstash/mapping.json:/mapping.json:ro"
+    depends_on:
+      - elasticsearch
     restart: always
-
   reporting-service:
     image: "zebrunner/reporting-service:${TAG_REPORTING_SERVICE}"
+    container_name: reporting-service
+    volumes:
+      - "assets-data:/opt/assets"
+    env_file:
+      - configuration/reporting-service/variables.env
     depends_on:
       - postgres
       - rabbitmq
       - redis
-    volumes:
-      - "./runtime/assets:/opt/assets"
-    env_file:
-      - configuration/reporting-service/variables.env
     restart: always
-
   reporting-ui:
     image: "zebrunner/reporting-ui:${TAG_REPORTING_UI}"
+    container_name: reporting-ui
+    environment:
+# https://github.com/zebrunner/zebrunner/issues/2127
+      - ZAFIRA_API_HOST=http://localhost:80
+      - ZAFIRA_API_CONTEXT_PATH=reporting-service
     depends_on:
       - reporting-service
-    env_file:
-      - configuration/reporting-ui/variables.env
     restart: always
+volumes:
```

Comment:
What is the benefit of using volumes in our particular case? Not asking about volumes in general, just train to understand what's the big benefit for us? Windows/Linux consistency?

Reply:
Another note - would it make sense to name volumes according to the names of services they belong? E.g. postgres volume: `postgres-data-volume`, `elasticsearch-data-volume`, `reporting-service-assets-volume` etc.

Reply:
1. using volumes we solve the sharing permission issues. No more requirements to start with uid and guid equals 1000. Also it should be easier to support Windows I guess.
2. by default docker add for each volume prefix, i.e. real names on the system:
reporting-service_assets-data
reporting-service_esdata-volume
reporting-service_pgdata-volume